Modules/Album/Source/Shared/Image/BusinessLogic/Model/Image.swift

```diff
@@ -1,6 +1,11 @@
 import Foundation
 
-struct Image {
+class Image {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/MarcoSantarossa/mssample-ios/blob/576acfc7a9586a705531c2483c3f000240f3fdc0/Modules/Album/Source/Shared/Image/BusinessLogic/Model/Image.swift#L3"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>